lib/index.js

```diff
@@ -40,7 +40,7 @@ class JsonDiff {
         if (!change.equal) {
           result[key] = change.result
           equal = false
-        } else if (this.options.full) {
+        } else if (this.options.full || (this.options.outputKeys && this.options.outputKeys.includes(key))) {
```

Comment:
This can be simply:
```
} else if (this.options.full || this.options.outputKeys.includes(key)) {
```
Since even an empty array is truthy, this test "this.options.outputKeys" doesn't really do anything. Ex:
```
> !![]
true
```

Reply:
The thing is that the tests call directly this function and those cases they don't build the entire `options` object, which leads to `outputKeys` being undefined. 
I will change the JsonDiff builder to always construct a valid instance (with an empty array if the outputKeys is not provided) and remove that check then.